internal/internal_workflow_testsuite_test.go

```diff
@@ -1741,10 +1767,16 @@ func (s *WorkflowTestSuiteUnitTest) Test_WorkflowLocalActivityWithMockAndListene
 
 	env.SetOnLocalActivityCompletedListener(func(activityInfo *ActivityInfo, result Value, err error) {
 		s.NoError(err)
-		var resultValue string
+		var resultValue *commonpb.Payload
```

Comment:
I am not sure if it is ok that `result` has `*commonpb.Payload` now. I don't actually understand how `string` worked here before. Was it implicitly converted from `[]byte`?

Reply:
It was using DataConverter in result.Get
This test should keep workfing without any changes.

Reply:
It seems like it is double encoded. Need to investigate.